glib/src/source.rs

```diff
@@ -827,44 +827,54 @@ where
 
 // rustdoc-stripper-ignore-next
 /// The priority of sources
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
-pub struct Priority(i32);
+#[derive(Default, Clone, Copy, Debug, Eq, PartialEq)]
```

Comment:
You probably also want to implement `PartialOrd` / `Ord` based on the integer values